backend/pkg/kubeconfig/kubeconfig.go

```diff
@@ -342,3 +354,52 @@ func LoadAndStoreKubeConfigs(kubeConfigStore ContextStore, kubeConfigs string, s
 
 	return nil
 }
+
+// getErrorChecks returns a list of error checks.
+func getErrorChecks() []errorCheck {
+	return []errorCheck{
+		{
+			Condition: func(e error) bool {
+				return strings.Contains(e.Error(), "failed to parse YAML in kubeconfig")
+			},
+			Message: "failed to parse YAML in kubeconfig",
+		},
+		{
+			Condition: func(e error) bool {
+				return strings.Contains(e.Error(), "cannot unmarshal number into Go struct field") &&
+					(strings.Contains(e.Error(), "client-certificate-data") ||
+						strings.Contains(e.Error(), "client-key-data") ||
+						strings.Contains(e.Error(), "certificate-authority-data"))
+			},
+			Message: "invalid certificate or key data values in kubeconfig",
+		},
+		{
+			Condition: func(e error) bool {
```

Comment:
when there is an error with `client-certificate-data` both the conditions pass and create an bit ambiguous message
<img width="707" alt="image" src="https://github.com/headlamp-k8s/headlamp/assets/22731013/51183bc5-533a-4e7a-9b43-c5088e240b3f">

Reply:
yes, the purpose was to append all the errors of the kubeconfig rather than the first hit. Users can have multiple user info in kubeconfig. So I appended those errors. WDYT?